basics/a54.go

```diff
@@ -0,0 +1,35 @@
+// tuple assignment x = y = y, x or multiple variable assignment
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var a, b ,c, d int
+	a = 10
+	b = 5
+	c = 25
+	d = 100
+	fmt.Println("Initial  value of a, b, c, d are ", a, b, c, d)
```

Comment:
fmt.Println add space, hence you don't need a space after 'are' here.

fmt.Println("Initial  value of a, b, c, d are", a, b, c, d)